src/components/navigation/Navigation.tsx

```diff
@@ -1,35 +1,42 @@
-import { Link } from "react-router-dom";
-import Button from "../common/Button";
+import { useState } from "react";
+import NavigationItem from "./NavigationItem";
+import { HomeIcon, CalendarDaysIcon, PlusCircleIcon, ChatBubbleOvalLeftEllipsisIcon, UserCircleIcon } from '@heroicons/react/20/solid'
+import PostsMenu from "./PostsMenu";
 
-// 홈 오늘의야구 커뮤니티 실시간채팅 알림 내정보
+const ICON_SIZE = "size-8"
+const LIST_STYLE = "py-4"
 
 const Navigation = () => {
+  const [isClickPostBtn, setIsClickPostBtn] = useState<boolean>(false);
+  
+  const handleClick = () => {
+    setIsClickPostBtn(!isClickPostBtn)
```

Comment:
이전 상태값을 이용해서 새로운 상태를 만들때는 callback을 이용해주세요!
(React에서는 updater function이라고 부르고 있어요!)
```suggestion
    setIsClickPostBtn(prev => !prev)
```